ouroboros-network/test/Test/Ouroboros/Network/PeerSelection.hs

```diff
@@ -678,30 +682,6 @@ prop_governor_gossip_1hr env@GovernorMockEnvironment {
                       "found:     " ++ show found) $
       property (found `Set.isSubsetOf` reachable)
 
-    -- We expect to find enough of them, either the target number or the
-    -- maximum reachable.
-    bigEnoughProperty found reachable
-        -- But there's an awkward corner case: if the number of public roots
-        -- available is bigger than the target then we will likely not get
-        -- all the roots (but which subset we get is random), but if we don't
-        -- get all the roots then the set of peers actually reachable is
-        -- incomplete, so we cannot expect to reach the usual target.
-        --
-        -- But we can at least expect to hit the target for root peers.
-      | Set.size (publicRootPeers `Set.union` localRootPeersSet)
-      > targetNumberOfRootPeers targets'
-      = property (Set.size found >= targetNumberOfRootPeers targets')
-
-      | otherwise
-      = counterexample ("reachable : " ++ show reachable ++ "\n" ++
-                        "found     : " ++ show found ++ "\n" ++
-                        "found #   : " ++ show (Set.size found) ++ "\n" ++
-                        "expected #: " ++ show expected) $
-        property (Set.size found == expected)
-      where
-        localRootPeersSet = LocalRootPeers.keysSet localRootPeers
-        expected = Set.size reachable `min` targetNumberOfKnownPeers targets'
-
```

Comment:
it's no longer true?

Reply:
It's not, since now with PeerSharing nodes might not be reachable via PeerSharing. It should be explained in the commit message, right?

Reply:
Yes, that would be helpful.

Reply:
I was asking if its not already. I believe it is.

Reply:
I see, yes it's fine.